mrSerious/StackOverflow-lite
# #160358197 Build front-end to consume the user profile end-point

#### What does this PR do?
Have a front-end to consume the user profile end-point
#### Description of Task to be completed?
Have a working front-end to consume the following endpoint
`GET /api/v1/users/:userId`
#### How should this be manually tested?
Check that front-end consumes the endpoint as follows:
- Open your browser
- Navigate [here](https://stack-overflow-lite-app.herokuapp.com/)
- Login and click the **My Profile** link OR click the username link
- A view with the user's information should now be open
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#160358197](https://www.pivotaltracker.com/n/projects/2189147/stories/160358197)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
